public/app/features/correlations/components/EmptyCorrelationsCTA.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
```

Comment:
Why with an expicit CTA suffix? Haven't seen it for any other call-to-action components.

Reply:
no real reason, i just used the component's name when importing given it's a default export.

as of why i'm exporting it as `EmptyCorrelationsCTA` it's kinda for the same reason.